apps/web/src/store/collateral/hooks.ts

```diff
@@ -49,6 +62,25 @@ export function useCollateralChangeCollateralType(): (collateralType: CurrencyKe
   return React.useCallback(
     (collateralType: CurrencyKey) => {
       dispatch(changeCollateralType({ collateralType }));
+      const defaultXChainId = DEFAULT_TOKEN_CHAIN[collateralType];
```

Comment:
add the function in `useCollateralActionHandlers` hook and change the function name to `changeCollateralType`

Reply:
moved to https://github.com/balancednetwork/balanced-network-interface/issues/1388